dockerfile/rocm4.2-pytorch1.7.0.dockerfile

```diff
@@ -41,6 +42,18 @@ RUN wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | APT_KEY_DON
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* /tmp/*
 
+# Install Open MPI
+ENV OPENMPI_VERSION=4.0.0
```

Comment:
use the same version 4.0.5 as cuda image?

Reply:
Done.